### ISSUE TITLE
unused types, false positive, via alias

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1698]
```abap
INTERFACE lif_bar.
  TYPES bar TYPE i.
ENDINTERFACE.

CLASS lcl DEFINITION.
  PUBLIC SECTION.
    INTERFACES lif_bar.
    ALIASES moo FOR lif_bar~bar.
    DATA foo TYPE moo.
ENDCLASS.

CLASS lcl IMPLEMENTATION.
ENDCLASS.
```

**Comment by larshp** [comment]
working, closing

![image](https://user-images.githubusercontent.com/5888506/163803450-47162d0d-9c5b-48b5-8d3f-b1fe1c4aa750.png)
